english/movie-ripoffs/hogwarts/burn-marshmallows

```diff
@@ -0,0 +1,5 @@
+It works! It actually works! You are a wizard!
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.